Everest03.NET/Validators/CustomersValidator.cs

```diff
@@ -0,0 +1,63 @@
+using FluentValidation;
+using System;
+
+namespace Everest03.NET.Validators
+{
+    public class CustomersValidator : AbstractValidator<Customer>
+    {
+        public CustomersValidator()
+        {
+            RuleFor(customer => customer.FullName)
+                .NotEmpty()
+                .MinimumLength(8);
+
+            RuleFor(customer => customer.Email)
+                .NotEmpty()
+                .EmailAddress()
+                .WithMessage("Email inv�lido, formata��o correta �: xxxxx@xxxxx.xxx");
+            
+            RuleFor(customer => customer)
+                .Must(customer => customer.Email == customer.EmailConfirmation)
+                .WithMessage("Email e EmailConfirmation precisam ser iguais");
+            
+            RuleFor(customer => customer.Cpf)
+                .NotEmpty()
+                .IsValidCPF()
+                .WithMessage("CPF inv�lido, formata��o correta �: 000.000.000-00");
```

Comment:
Não é boa prática backend esperar receber valor mascarado, deve aceitar mas não obrigar
```suggestion
                .WithMessage("CPF inv�lido, formato esperado: 000.000.000-00");
```